tests/test_numbers.py

```diff
@@ -252,6 +253,13 @@ def test_format_currency():
     assert (numbers.format_currency(1099.98, 'EUR', locale='nl_NL')
             != numbers.format_currency(-1099.98, 'EUR', locale='nl_NL'))
 
+    assert (numbers.format_currency(1099.98, 'JPY', locale='en_US',
+                                    currency_digits=True)
+            == u'\xa51,100')
+    assert (numbers.format_currency(1099.98, 'COP', u'#,##0.00', locale='es_ES',
```

Comment:
There're no tests with `currency_digits=False`. I'm assuming that `current_digits=None` is the default and current tests are doing their jobs to ensure the correct behaviour.